src/makeCalculator.js

```diff
@@ -38,6 +38,41 @@
  */
 function makeCalculator() {
   // write code here
+  return {
+    result: 0,
+
+    add(a, b) {
+      return a + b;
+    },
+
+    subtract(a, b) {
+      return a - b;
+    },
+
+    multiply(a, b) {
+      return a * b;
+    },
+
+    divide(a, b) {
+      if (b === 0) {
+        throw new Error(`Invalid value`);
```

Comment:
Error message should be clear about what's wrong.
```suggestion
        throw new Error(`Division by zero`);
```